projects/assets/src/translations/en/payment.ts

```diff
@@ -14,7 +14,7 @@ export const payment = {
     saveAsDefault: 'Save as default',
     setAsDefault: 'Set as default payment method',
     billingAddress: 'Billing address',
-    sameAsShippingAddress: 'Same as shipping address',
+    sameAsDeliveryAddress: 'Same as shipping address',
```

Comment:
if we're going through with this https://github.com/SAP/spartacus/pull/14828/files#r790410745 (notably slack)

Reply:
I was thinking from the accelerator SS that I showed you. Most things are delivery-address, but labels are Same as shipping address. Should we do it like that?

Reply:
as mentioned earlier today since we won't do it like accelerator where path shows delivery-address, but label shows shipping-address, I changed all references for those labels and value to have Delivery instead of Shipping.

In the e2e, I did not update the methods and variables because I find it redundant fixing that 'now' haha, but I did change those references/variables in our checkout lib (except old checkout since we're dropping that)